src/Language/PureScript/Docs/Prim.hs

```diff
@@ -258,12 +258,20 @@ kindSymbol = primType "Symbol" $ T.unlines
   , ""
   , "Construct types of this kind using the same literal syntax as documented"
   , "for strings."
+  , ""
+  , "    type Hello :: Symbol {- String -}"
+  , "    type Hello = \"Hello, world\""
+  , ""
   ]
 
 kindRow :: Declaration
 kindRow = primType "Row" $ T.unlines
   [ "`Row` is the kind constructor of label-indexed types which map type-level strings to other types."
-  , "For example, the kind of `Record` is `Row Type -> Type`, mapping field names to values."
+  , "For example..."
+  , ""
+  , "     type ExampleRow = ( label :: TypeName, name :: String )"
+  , ""
+  , "`Row`s are often used in `Record`, whose kind signature is `Row Type -> Type`, mapping field names to values."
```

Comment:
```suggestion
  , "The most common use of `Row` is `Row Type`, a row mapping labels to basic (of kind `Type`) types:"
  , ""
  , "    type ExampleRow :: Row Type"
  , "    type ExampleRow = ( name :: String, values :: Array Int )"
  , ""
  , "This is the kind of `Row` expected by the `Record` type constructor."
  , "More advanced row kinds like `Row (Type -> Type)` are used much less frequently."
```

Reply:
I've pushed this change. Can I get an approval, so we can merge this?